partest-suite/src/test/resources/scala/tools/partest/scalajs/2.12.4/run/anyval-box-types.check

```diff
@@ -0,0 +1,52 @@
+true
+1
+true
+1
+true
+-1
+true
+1
+true
+false
+true
+true
```

Comment:
The 2 lines above (11 and 12) are `false` in the original .check file, because they test `1.isInstanceOf[Short]` and `1.isInstanceOf[Double]`, which is false on the JVM but true in Scala.js.

[Source of the test](https://github.com/scala/scala/blob/3b7623facb6ce48c28d6afdf84e74a99d45f0e23/test/files/run/anyval-box-types.scala#L14-L15)